### PR TITLE
feat(feedback): add feedback issue creation

### PR DESCRIPTION
**Pull request by JoshFerge**
- Adds feedback issue creation. We'll create both a feedback object and a feedback issue. We may not need a feedback object, but will create it for now until the retrieval endpoints support querying the new data.

ref: https://github.com/getsentry/team-replay/issues/225

### CODE REVIEW

File: src/sentry/feedback/usecases/create_feedback.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    evidcence_data, evidence_display = make_evidence(event["feedback"])
+    occurrence = IssueOccurrence(
+        id=uuid4().hex,
+        event_id=event.get("event_id") or uuid4().hex,
+        project_id=project_id,
+        fingerprint=[
+            uuid4().hex
+        ],  # random UUID for fingerprint so feedbacks are grouped individually
+        issue_title="User Feedback",
+        subtitle=event["feedback"]["message"],
```
Comment (cmanallen) [inline src/sentry/feedback/usecases/create_feedback.py:37]: Should this be truncated?

File: src/sentry/feedback/usecases/create_feedback.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    event["event_id"] = event.get("event_id") or uuid4().hex
+    evidcence_data, evidence_display = make_evidence(event["feedback"])
+    occurrence = IssueOccurrence(
+        id=uuid4().hex,
+        event_id=event.get("event_id") or uuid4().hex,
+        project_id=project_id,
+        fingerprint=[
+            uuid4().hex
+        ],  # random UUID for fingerprint so feedbacks are grouped individually
+        issue_title="User Feedback",
```
Comment (cmanallen) [inline src/sentry/feedback/usecases/create_feedback.py:36]: How will this be displayed?  Will a user see "User Feedback" repeated over and over again?  Maybe a better system would be to have the message here and the subtitle be the location of the feedback.  I'm not sure "User Feedback" is a meaningful title if the page is dedicated to user feedback.

File: src/sentry/feedback/usecases/create_feedback.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        subtitle=event["feedback"]["message"],
+        resource_id=None,
+        evidence_data=evidcence_data,
+        evidence_display=evidence_display,
+        type=FeedbackGroup,
+        detection_time=ensure_aware(datetime.datetime.fromtimestamp(event["timestamp"])),
+        culprit="user",  # TODO: fill in culprit correctly -- URL or paramaterized route/tx name?
+        level="info",  # TODO: severity based on input?
+    )
+    event_data = {"fingerprint": occurrence.fingerprint, "project_id": project_id, **event}
```
Comment (cmanallen) [inline src/sentry/feedback/usecases/create_feedback.py:46]: The call-site does not include fields such as "environment" and "platform" in the event kwargs.  Are those fields important?

Does "fingerprint" need to be repeated here?  I thought it was only specified in the "IssueOccurence".

File: src/sentry/feedback/usecases/create_feedback.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        ],  # random UUID for fingerprint so feedbacks are grouped individually
+        issue_title="User Feedback",
+        subtitle=event["feedback"]["message"],
+        resource_id=None,
+        evidence_data=evidcence_data,
+        evidence_display=evidence_display,
+        type=FeedbackGroup,
+        detection_time=ensure_aware(datetime.datetime.fromtimestamp(event["timestamp"])),
+        culprit="user",  # TODO: fill in culprit correctly -- URL or paramaterized route/tx name?
+        level="info",  # TODO: severity based on input?
```
Comment (cmanallen) [inline src/sentry/feedback/usecases/create_feedback.py:44]: Just pointing out there are two todos here.  You don't need to resolve these to merge 👍

**Review by cmanallen — APPROVED** [review]

File: src/sentry/feedback/usecases/create_feedback.py (hunk 1 above)
Comment (JoshFerge) [reply src/sentry/feedback/usecases/create_feedback.py:37]: Truncation/validation will be handled upstream -- if this ends up throwing here, we can add safeguards.

File: src/sentry/feedback/usecases/create_feedback.py (hunk 2 above)
Comment (JoshFerge) [reply src/sentry/feedback/usecases/create_feedback.py:36]: This is a good point. I'll add a comment here. The intention is for these "Issues" to be hidden in the issues stream UI eventually, while our Feedback UI will display all the information correctly, the "Title" isn't super important atm.

File: src/sentry/feedback/usecases/create_feedback.py (hunk 3 above)
Comment (JoshFerge) [reply src/sentry/feedback/usecases/create_feedback.py:46]: for fingerprint, was copying from here: https://github.com/getsentry/sentry/blob/713cd3053fa608fd1a549975abdc2a2225a62cdf/src/sentry/monitors/logic/mark_failed.py#L296

and I believe that `environment` and `platform` should be in `event`.

File: src/sentry/feedback/usecases/create_feedback.py (hunk 4 above)
Comment (JoshFerge) [reply src/sentry/feedback/usecases/create_feedback.py:44]: yep. can see what ends up looking best / how we can fill in the data.

File: src/sentry/feedback/usecases/create_feedback.py (hunk 2 above)
Comment (JoshFerge) [reply src/sentry/feedback/usecases/create_feedback.py:39]: comment added!

File: src/sentry/feedback/usecases/create_feedback.py (hunk 1 above)
Comment (JoshFerge) [reply src/sentry/feedback/usecases/create_feedback.py:40]: And on subtitle ends up being too long, we can truncate here.

File: src/sentry/feedback/usecases/create_feedback.py (hunk 3 above)
Comment (cmanallen) [reply src/sentry/feedback/usecases/create_feedback.py:49]: I misunderstood the function signature.  This looks right.

**Review by cmanallen — APPROVED** [review]
Looks good to me.

**Pull request merged**